For projects that using Amazon S3 and storage a large amount of images GLOB function can cause performance problem

diff --git a/Block/Adminhtml/Wysiwyg/Images/Tree.php b/Block/Adminhtml/Wysiwyg/Images/Tree.php
index 41e9358..f4456f0 100644
--- a/Block/Adminhtml/Wysiwyg/Images/Tree.php
+++ b/Block/Adminhtml/Wysiwyg/Images/Tree.php
@@ -76,7 +76,7 @@ class Tree extends \Magento\Backend\Block\Template
                 'cls' => 'folder',
             ];

-            $hasNestedDirectories = count(glob($item->getFilename() . '/*', GLOB_ONLYDIR)) > 0;
+            $hasNestedDirectories = true;

             // if no nested directories inside dir, add 'leaf' state so that jstree hides dropdown arrow next to dir
             if (!$hasNestedDirectories) {
